pages/reference/base-images/base-images.md

```diff
@@ -10,67 +10,66 @@ excerpt: Docker images maintained by {{ $names.company.lower }}
 
-- `<lang_ver>` is the version of the language stack, for example node.js 10.10, it can also be substituted for `latest`.
-- `<distro_ver>` is the version of the linux distro, for example in the case of Debian there are 4 valid versions, namely `sid`, `jessie`, `buster` and `stretch`.
-- For each combination of distro and stack we have two variants called `run` and `build`. The build variant is much heavier as it has a number of tools preinstalled to help with building source code. You can see an example of the tools that are included in the Debian Stretch variant [here](https://github.com/balena-io-library/base-images/blob/master/balena-base-images/armv7hf/debian/stretch/build/Dockerfile). The `run` variants are stripped down and only include a few useful runtime tools, see an example [here](https://github.com/balena-io-library/base-images/blob/master/balena-base-images/armv7hf/debian/stretch/run/Dockerfile).
-If no variant is specified, the image defaults to `run`
-- <yyyymmdd> the last optional field on tags is the date tag. This is useful for production deployments as these base images are non-moving tags, so no packages in these will update ever.
+- `<lang_ver>` is the version of the language stack, for example, Node.js 10.10, it can also be substituted for `latest`.
+- `<distro_ver>` is the version of the Linux distro, for example in the case of Debian, there are 4 valid versions, namely `sid`, `jessie`, `buster` and `stretch`.
+- For each combination of distro and stack, we have two variants called `run` and `build`. The build variant is much heavier as it has a number of tools preinstalled to help with building source code. You can see an example of the tools that are included in the Debian Stretch variant [here]({{ $links.githubLibrary }}/base-images/blob/master/balena-base-images/armv7hf/debian/stretch/build/Dockerfile). The `run` variants are stripped down and only include a few useful runtime tools, see an example [here]({{ $links.githubLibrary }}/base-images/blob/master/balena-base-images/armv7hf/debian/stretch/run/Dockerfile). If no variant is specified, the image defaults to `run`
+- The last optional field on tags is the date tag `<yyyymmdd>`. This is useful for production deployments as these base images are non-moving tags, so no packages in these will update ever.
 
 #### Examples
 
 `balenalib/raspberrypi3-node:10.17`
 
-- `<hw>` : raspberrypi3 - The raspberry pi 3 device type.
+- `<hw>` : raspberrypi3 - The Raspberry Pi 3 device type.
 - `<distro>` : omitted, so it defaults to Debian.
 - `<lang>` : node - the Node.js runtime and npm will be installed
-- `<lang_ver>` : 10.17 - This gives us node.js version 10.17.x whatever is the latest patch version provided on balenalib
-- `<distro_ver>` : omitted, so it defaults to `stretch`
+- `<lang_ver>` : 10.17 - This gives us Node.js version 10.18.x whatever is the latest patch version provided on balenalib
```

Comment:
Huh? Mix of 10.17 and 10.18

Reply:
Nice catch, which is why I want to autogenerate this as much as possible 😄